keel-sql/src/main/kotlin/com/netflix/spinnaker/keel/sql/SqlArtifactRepository.kt

```diff
@@ -57,7 +57,7 @@ class SqlArtifactRepository(
         .where(DELIVERY_ARTIFACT.UID.eq(DELIVERY_ARTIFACT_VERSION.DELIVERY_ARTIFACT_UID))
         .and(DELIVERY_ARTIFACT.NAME.eq(artifact.name))
         .and(DELIVERY_ARTIFACT.TYPE.eq(artifact.type.name))
-        .orderBy(DELIVERY_ARTIFACT_VERSION.VERSION.desc())
+        .orderBy(DELIVERY_ARTIFACT_VERSION.VERSION.desc()) // TODO: this is not going to work
```

Comment:
is this a future todo or something you wanted to address in this PR?

Reply:
Future. Semver strings are not alphabetically sortable so I don't think this is a great way to get "latest version first". It's kind of an orthogonal concern to this PR, though.